cypress/page/login.page.ts

```diff
@@ -0,0 +1,29 @@
+class LoginPage {
+  private emailSelector: string;
+  private emailUser: string;
+  private passwordSelector: string;
+  private passwordUser: string;
+  private signIn: string;
+
+  constructor () {
+    this.emailSelector = "#email";
+    this.emailUser = "aperdomobo@gmail.com";
+    this.passwordSelector = "#passwd";
+    this.passwordUser = "WorkshopProtractor";
```

Comment:
instead of use the email and password values here, you can add them as parameters in the login function and add the values in the test file when call the function :)